src/main/java/com/parasoft/findings/jenkins/coverage/api/metrics/source/CoverageSourcePrinter.java

```diff
@@ -82,15 +82,15 @@ public String getTooltip(final int line) {
         var missed = getMissed(line);
         if (covered + missed > 1) {
             if (missed == 0) {
-                return "All branches covered";
+                return Messages.All_Branches_Covered();
```

Comment:
The localization for source code view tooltip here is not working as expected, and Chinese properties entries are missing.

Reply:
We can set some placeholder text in `com.parasoft.findings.jenkins.coverage.api.metrics.source.CoverageSourcePrinter#getTooltip` when the source code is being processed into HTML, then replace the placeholder text with the localized text from `Messages` in `com.parasoft.findings.jenkins.coverage.api.metrics.source.SourceViewModel#getSourceFileContent`.